client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Looks like a default pre-written README, which is fine, but I think some of it may not appl,y to our app? The fact that `npm start` is not actually a dev build, for example.

Reply:
Yep it is pre-written README but `npm start` is dev build for React app though. I think this README works well for me but will consider update this later if there is anything wrong with it.

Reply:
Makes sense to me :)